doc/nrf/ug_zigbee.rst

```diff
@@ -0,0 +1,137 @@
+.. _ug_zigbee:
+
+Working with Zigbee
+###################
+
+The |NCS| provides support for developing applications using the Zigbee protocol.
+
+.. _zigbee_ug_intro:
+
+Introduction
+************
+
+Zigbee is a portable, low-power software networking protocol that provides connectivity over 802.15.4-based mesh network.
+It also defines an application layer that provides interoperability among all vendors.
+
+In combination with Zephyr RTOS and |NCS|, Zigbee allows for easy development of low-power connected solutions.
+
+.. _zigbee_ug_supported features:
+
+Supported features
+******************
+
+|zboss_lib|
+It includes all mandatory features of the |zigbee_version| specification and provides an Application Programming Interface to access different services.
+The stack comes with the following features:
+
+* Complete implementation of the Zigbee core specification and Zigbee Pro feature set.
+* Support for all device roles: coordinator, router, and end device.
+* Zigbee Cluster Library (a specification for end devices).
+* Base Device Behavior.
+* Devices, described in former Zigbee Home Automation and Light Link profiles.
+* Zigbee Green Power.
+* Experimental support for ``ZB_ZCL_WWAH``.
+
+Required libraries and drivers
+******************************
+
+|zboss_lib|
+This library is available in `nrfxlib`_.
+
+The OSIF subsystem acts as the linking layer between the ZBOSS stack and |NCS|.
```

Comment:
Documentation for this library is probably planned for after the release?

Reply:
As far as I know, this one is outside of NCS. I'll check with @tomchy .

Reply:
This PR adds  subsys/zigbee/osif/ - so if it isn't the full code, there's at least some integration libraries.

Reply:
Will add mention of the files for now, and rework this paragraph in the future, based on the discussion with the Zigbee team.